deploy.ci.functional.mariadb.yml

```diff
@@ -24,7 +24,7 @@ image:
 
 composer:
     mode: '--no-dev --quiet -o -a'
-    autoload: --classmap-authoritative
+    autoload: '-o -a --apcu'
```

Comment:
In the PR https://github.com/spryker/suite-nonsplit/pull/9948/files#diff-4af50962d01a77ee79dcbdb55596001d0e003ed7b9db315ee90dcd0e39104db3R31 this was made empty.

Can you please align with @aleksandr-velikanov ?

Reply:
What does it mean "it will be rewritten"? Can you clarfiy with @aleksandr-velikanov please how it must be now?

Reply:
it means that after switching to github CI I'll adjust CI for suite nonsplit.  nothing to align with him